src/common/ExportSettingsModal/ExportSettingsModal.js

```diff
@@ -75,7 +165,11 @@ const ExportSettingsModal = ({
 ExportSettingsModal.propTypes = {
   onCancel: PropTypes.func.isRequired,
   isExporting: PropTypes.bool.isRequired,
-  onExportCSV: PropTypes.func.isRequired,
+  handleSubmit: PropTypes.func.isRequired,
+  values: PropTypes.object.isRequired,
 };
 
-export default ExportSettingsModal;
+export default stripesFinalForm({
+  navigationCheck: true,
+  subscription: { values: true },
```

Comment:
IMHO, it's not a form at all, we don't make CRUD, don't send anything to server. Why not just use MultiSelection?